docs/xml/metainfo-component.xml

```diff
@@ -362,6 +362,15 @@
 
 				</listitem>
 			</varlistentry>
+
+			<varlistentry>
+				<term>source</term>
+				<listitem>
+				<para>
+					Should provide a web link to the source code.
```

Comment:
It would be nice to give an example here at least, as in "link to the source code, for example an online version control system browser."